docs/content/3.middleware/2.guides/9.kubernetes-probes.md

```diff
@@ -0,0 +1,130 @@
+# Kubernetes Probes
```

Comment:
Please check the document for grammar and spelling for ex. in Grammarly. I have found few.